README.md

```diff
@@ -131,5 +118,3 @@ App Service Plan: ASP-Cohort22PauCurProjectExercise-930f
 Web App: pdc-todo-app
 
 Azure App Service URL: https://pdc-todo-app.azurewebsites.net
-
-
```

Comment:
You should add a section about MongoDB and your environment variable setup. In particular, the reader should know which environment variable they need to populate and where to get the value from.

Reply:
Also, if you have a Trello section, make sure it's deleted/modified to reflect migration to Mongo DB